packages/kit/src/runtime/server/page/render.js

```diff
@@ -248,6 +248,7 @@ export async function render_response({
 		}
 
 		attributes.unshift('rel="stylesheet"');
+		head += `\n\t<link rel="preload" href="${path}" as="style">`;
```

Comment:
I think this preloading is already being done by the Link header. Maybe the header should be replaced with this (might benefit prerendered pages)? But only if it's not inlined, I guess.

Reply:
> I think this preloading is already being done by the Link header. Maybe the header should be replaced with this (might benefit prerendered pages)? But only if it's not inlined, I guess.

Does adding both have any drawbacks ?

Reply:
> Maybe the header should be replaced with this (might benefit prerendered pages)?

The header is better performance. We fallback to including in the page when prerendering: https://github.com/sveltejs/kit/blob/56cb62dcbad93b6317f98aa93c87f469f0b199eb/packages/kit/src/runtime/server/page/render.js#L260